docs/user-guide/glossary/edit-glossaries.md

```diff
@@ -1,6 +1,28 @@
 ### Prerequisite
 See [glossary roles and permission](/user-guide/glossary/glossary-roles-permissions) for permission setup.
 
+### From Glossary page
```

Comment:
These headings are hard to understand in this context - maybe just because I can't see the main heading on the page?